### DISCUSSION TITLE
Does the watchOS app provide more data such as percentage/position

### DISCUSSION THREAD

**Discussion by jaywick** [TortugaPower/BookPlayer#754] [Q&A]
It looks like the watchOS app supports [corner complications](https://developer.apple.com/documentation/clockkit/graphic/corner_complication_templates) but all I see is the app icon and `----`.

From list

![image](https://user-images.githubusercontent.com/3955052/151910634-6070858f-62b3-409e-9c6d-2930e36dc739.png)

When added
![image](https://user-images.githubusercontent.com/3955052/151910725-23ab07b4-f206-47b4-b01e-67e19c9a16fa.png)

Not sure if it's a bug, or not implemented. If the latter, wondering if there's a task to implement it, it would be nice to contribute to this project!

**Comment by GianniCarlo** [comment]
This is addressed in the beta we launched today (link to join: https://testflight.apple.com/join/FLv3zLyN). For this complication we opted to put the current chapter index, and the full progress of the book as a progress bar (not much space to put words in there :/). My apple watch is a series 3, so I cannot test how well it'll behave, but we'll do a long beta for this one, we plan to include other things too in 4.5.0 👍
